hudi-utilities/src/main/java/org/apache/hudi/utilities/schema/SchemaRegistryProvider.java

```diff
@@ -193,8 +201,8 @@ public Schema getTargetSchema() {
     String targetRegistryUrl = config.getString(HoodieSchemaProviderConfig.TARGET_SCHEMA_REGISTRY_URL.key(), registryUrl);
     try {
       return parseSchemaFromRegistry(targetRegistryUrl);
-    } catch (IOException ioe) {
-      throw new HoodieIOException("Error reading target schema from registry :" + registryUrl, ioe);
+    } catch (Exception e) {
```

Comment:
shouldn't we log targetRegistryUrl instead of registryUrl

Reply:
yep